### PR TITLE
Implement course page

### PR DESCRIPTION
**Pull request by DeanCivlin**
**Added Course Page** 

1. Created new html&css course-page files
2. Added relevant Views.py func
3. Added relevant code to urls.py
4. Removed redundant text-center class from app_layout
5. Added Test to test_client.py

### CODE REVIEW

**Comment by DeanCivlin** [comment]
Need to implement context tests here as well. will do soon

**Review by razashtar93 — APPROVED** [review]
Great job Dean,
just finish tests and all LGTM

File: AcadeMeData/tests/test_views.py [hunk 1]
```diff
@@ -20,3 +20,13 @@ def test_contactus_html(client, user_example):
     assertTemplateUsed(response, 'landing/contact_us.html')
     assert response.context['navbar_links'] == {f"Welcome {user_example.user.username}": "#",
                                                 "Logout": "/logout"}
+
+
+@pytest.mark.django_db
+def test_course_html(client, user_example):
```
Comment (DeanCivlin) [inline AcadeMeData/tests/test_views.py:26]: @Yarboa Here is the test I added to test the course page context

**Review by Yarboa — COMMENTED** [review]
LGTM please rebase

**Comment by DeanCivlin** [comment]
@Yarboa 
Rebase is done.

**Review by DanielItkes — APPROVED** [review]
Good work Dean!

**Review by Yarboa — APPROVED** [review]

**Pull request merged**